src/main/resources/application.yaml

```diff
@@ -60,7 +60,7 @@ whatsapp:
   api:
     phone_number_id: ${whatsapp_api_phone_number_id:341358742400656}
     url: https://graph.facebook.com/v19.0
-    token: ${whatsapp_api_token}
+    token: ${whatsapp_api_token:fcajouhdfqiuohdawihda}
```

Comment:
remove the default value here, use the application-local.yaml if you need to sent dummy api_token